app/assets/stylesheets/sdg/help.scss

```diff
@@ -0,0 +1,29 @@
+.sdg-help-content {
+  @include grid-column-gutter;
+
+  > .tabs {
+    $spacing: 0.5%;
+
+    li {
+      min-width: $sdg-icon-min-width;
+      width: calc(100% / 17 - #{$spacing});
+      margin-right: $spacing;
+
+      &:focus {
+        outline: $outline-focus;
+      }
+
+      &:hover {
+        cursor: pointer;
+      }
+
+      a[aria-selected=true] img {
```

Comment:
Pronto reports:

> Avoid qualifying attribute selectors with an element

Translation: don't mix `a` with `[aria-selected]` on the same selector.

I'm not so fond of this rule, but since it's there, we might as well follow it until (if) we decide to remove it :thinking: